.cirrus.yml

```diff
@@ -53,13 +57,13 @@ build_task:
   env:
     GITHUB_TOKEN: ENCRYPTED[!f458126aa9ed2ac526f220c5acb51dd9cc255726b34761a56fc78d4294c11089502a882888cef0ca7dd4085e72e611a5!]
```

Comment:
To remove this `GITHUB_TOKEN` need to trigger build of `.cirrus/python-env.Dockerfile`.